src/methods/defining_methods/hard/q3/MainTest.java

```diff
@@ -0,0 +1,69 @@
+package methods.defining_methods.hard.q3;
+
+import java.util.stream.Stream;
+
+import global.MethodTest;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.utils.MethodUtil;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+public class MainTest extends BaseTest {
+    // Java
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "8";
+
+        return new Clause[]{
+                new StringLiteral("Enter the number of sides on the polygon:"),
+                new NewLine(),
+                new PlaceHolder()
+        };
+    }
+
+    public void runMain() {
+        Polygo.main(new String[0]);
+    }
+
+    static Stream<Arguments> innerAnglesInputProvider() {
+        return Stream.of(
+                Arguments.of(8, 1080),
+                Arguments.of(3, 180),
+                Arguments.of(100, 17640),
+                Arguments.of(10000, 1799640)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("innerAnglesInputProvider")
+    void correctInnerAnglesMethod(int a, int innerAngles) throws Throwable {
+        Object[][] arguments = {
+                {a, int.class}
+        };
+        MethodTest m = new MethodTest(Polygo.class, "innerAngles", arguments);
+        Object output = m.callMethod();
+        String errorMessage = "Your innerAngles() method does not correctly calculated the inner angles of a polygon.";
```

Comment:
Should be calculate

Reply:
Also, say something about the sum of the inner angles in degrees (so they know not to use radians)